updateStableIds/src/main/java/org/reactome/release/update_stable_ids/DatabaseUtils.java

```diff
@@ -0,0 +1,10 @@
+package org.reactome.release.update_stable_ids;
+
+public class DatabaseUtils {
```

Comment:
It doesn't look like this class is doing anything useful. Please remove it or make it more useful.